SwiftSyntaxCLI/Package.swift

```diff
@@ -0,0 +1,53 @@
+// swift-tools-version:5.7
+
+import PackageDescription
+import Foundation
+
+let package = Package(
+  name: "SwiftSyntaxCLI",
+  platforms: [
+    .macOS(.v10_15)
+  ],
+  products: [
+    .executable(name: "swift-syntax-cli", targets: ["swift-syntax-cli"])
+  ],
+  dependencies: [
+    .package(path: "..")
+  ],
+  targets: [
+    .target(
+      name: "InstructionCounter"
+    ),
+
+    .executableTarget(
+      name: "swift-parser-cli",
+      dependencies: [
+        "InstructionCounter",
+        .product(name: "ArgumentParser", package: "swift-argument-parser"),
+        .product(name: "SwiftDiagnostics", package: "swift-syntax"),
+        .product(name: "SwiftOperators", package: "swift-syntax"),
+        .product(name: "SwiftParser", package: "swift-syntax"),
+        .product(name: "SwiftParserDiagnostics", package: "swift-syntax"),
+        .product(name: "SwiftSyntax", package: "swift-syntax"),
+      ]
+    ),
```

Comment:
@ahoppen should we have a single executable or two? 

We can split it as I've done with the build an test etc.